### PR TITLE
vim_cheat_sheet: Add ex-commands and registers section

### PR DESCRIPTION
**Pull request by NaveenKarippai**
- [ ] New Instant Answer
    - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [x] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [x] Enhancement: **`vim_cheat_sheet: added more sections`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

As a vim user, registers make the life easier to get the daily
programming tasks done. Additional ex-commands cheat is also handy
as ex-editor is at the core of vim editor.

Provide an overview of the changes this pull request introduces.

* added ex-commands section

* added registers section

###### People to notify (@GuiltyDolphin , @moollaza )

---

Instant Answer Page: https://duck.co/ia/view/vim_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @NaveenKarippai

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/vim.json [hunk 1]
```diff
@@ -27,7 +27,8 @@
         "Visual Commands",
         "Working with Multiple Files",
         "Tabs",
-        "Miscellaneous",
+	"Registers",
```
Comment (gaulrobe) [inline share/goodie/cheat_sheets/json/vim.json]: Need an extra space here.

**Comment by NaveenKarippai** [comment]
Thanks! LGTM

**Comment by GuiltyDolphin** [comment]
@NaveenKarippai Thanks :smile:

**Pull request merged**